changelog/radek_add-stateId-param.md

```diff
@@ -0,0 +1,3 @@
+### Fixed
+
+- Add `{state_id}` to Prysm endpoints.
```

Comment:
Please add more to this for the changelog. This is a breaking change for API paths and we should communicate how each of them have changed.